src/components/Icon.jsx

```diff
@@ -1,5 +1,14 @@
+import React from "react";
+import Image from "next/image";
+
 const Icon = () => {
-  return <div>Icon</div>;
+  return (
+    <div className="container">
+      <div className="image">
+        <Image src="navbar-logo.svg" alt="Navbar-Logo" width={57} height={64} />
+      </div>
```

Comment:
include the text "HLG" next to the icon

check out the picture from the issue for reference

Reply:
![image](https://github.com/acm-ucr/hlg-website/assets/54856436/2a6ac33e-c734-4473-992a-fcc4dd5439f8)
added text, made icon as object, and made text thicc